components/molecules/InfoCard/InfoCard.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Link } from "../../atoms";
+import { IconArrowForward } from "../../icons";
+
+import { InfoCard, InfoCardProps } from "./InfoCard";
+
+export default {
+  title: "Design System/Molecules/InfoCard",
```

Comment:
nit: 
```suggestion
  title: "DesignSystem/Molecules/InfoCard",
```

Reply:
@xenowits this is ok with the space, this correspond to the folder structure 
<img width="335" alt="image" src="https://user-images.githubusercontent.com/5679878/168752195-b7ce8850-3a3f-4731-85b0-905138bb145f.png">